goal_src/levels/village2/swamp-blimp.gc

```diff
@@ -5,3 +5,2403 @@
+                                            )
+                                           "sksp0157"
+                                           (the-as entity #f)
+                                           *entity-pool*
+                                           (game-task none)
+                                           )
+                                          )
+                                         ((= arg0 2)
+                                          (level-hint-spawn
+                                           (game-text-id
+                                            swamp-tethers-two-to-go
+                                            )
+                                           "sksp0158"
+                                           (the-as entity #f)
+                                           *entity-pool*
+                                           (game-task none)
+                                           )
+                                          )
+                                         ((= arg0 1)
+                                          (level-hint-spawn
+                                           (game-text-id
+                                            swamp-tethers-lefts-find-the-last
+                                            )
+                                           "sksp0159"
+                                           (the-as entity #f)
+                                           *entity-pool*
+                                           (game-task none)
+                                           )
+                                          )
+                                         ((zero? arg0)
+                                          (level-hint-spawn
+                                           (game-text-id
+                                            swamp-tethers-completion-sage-precursor-arm
+                                            )
+                                           "sagevb04"
+                                           (the-as entity #f)
+                                           *entity-pool*
+                                           (game-task none)
+                                           )
+                                          )
+                                         )
+                                        (none)
+                                        )
+         s5-2
+         )
+        (-> gp-3 ppointer)
+        )
+       )
+      )
+     )
+    )
+   (go swamp-tetherrock-hide)
+   (none)
+   )
+  :post
+  (the-as (function none :behavior swamp-tetherrock) ja-post)
+  )
+
+(defstate swamp-tetherrock-idle (swamp-tetherrock)
+  :event
+  (behavior ((arg0 process) (arg1 int) (arg2 symbol) (arg3 event-message-block))
+   (case arg2 
+    (('attack)
+      (when (and (>= arg1 2) (= (-> arg3 param 1) 'eco-yellow))
+       (+! (-> self hits) 1)
+       (cond
+        ((>= (-> self hits) 1)
```

Comment:
interesting, it looks like it might have taken > 1 hit to destroy the boulders at some point.